calico-enterprise/threat/configuring-webhooks.mdx

```diff
@@ -5,25 +5,27 @@ title: Webhooks for security events
 
 # Webhooks for security event alerts
 
-You can configure $[prodname] webhooks to post security alerts directly to Slack, Jira, or any custom HTTP endpoint.
+You can configure $[prodname] webhooks to post security alerts directly to Slack, Jira, AlertManager, or any custom HTTP endpoint.
 
 ## Before you begin
 
 Your target application must be configured to receive data from the $[prodname] webhook.
 
 * **Slack**. You must have a webhook URL for the Slack app that you want $[prodname] to send alerts to.
-See [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks) for more information.
+  See [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks) for more information.
 * **Jira**. You must have an API token for an Atlassian user account that has write permissions to your Jira instance.
   See [Manage API tokens for your Atlassian account](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/) for details on how to obtain an API token.
   You also need:
   * Your Atlassian site URL. If you access Jira at the URL `https://<your-company>.atlassian.net/jira`, then your site URL is `<your-company>.atlassian.net`.
   * A Jira project key. This is the Jira project where your $[prodname] webhook creates new issues. This user associated with your API token must have write permissions to this project.
+* **AlertManager** You must have a webhook URL for your AlertManager instance to send alerts to. Optionally you can also configure authentication if needed.
+  See [AlertManager | Prometheus] (https://prometheus.io/docs/alerting/latest/alertmanager/) for more information.
```

Comment:
TBD -- see above, may need to be more specific for finding or generating he webhook URL.

Reply:
Done.